consensus/polybft/contractsapi/contractsapi.go

```diff
@@ -0,0 +1,91 @@
+// Code generated by scapi/gen. DO NOT EDIT.
+package contractsapi
+
+import (
+	"math/big"
+
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/umbracle/ethgo"
+	"github.com/umbracle/ethgo/abi"
+)
+
+var (
+	_ = big.NewInt
+	_ = types.Address{}
+	_ = ethgo.Log{}
+)
+
+var (
+	commitType = abi.MustNewMethod("function commit(tuple(uint256 startId,uint256 endId,uint256 leaves,bytes32 root) bundle,bytes signature,bytes bitmap)") //nolint:all
```

Comment:
I suggest to add an exception for generated files instead of holing

Reply:
Not sure I follow what do you mean?